user_guide_src/source/libraries/validation.rst

```diff
@@ -201,6 +201,42 @@ Then add validation rules in the controller (**Form.php**)::
 
 If you submit the form you should see the success page or the form with error messages.
 
+Config for Validation
+*********************
+
+.. _validation-traditional-and-strict-rules:
+
+Traditional and Strict Rules
+============================
+
+CI4 has two kind of Validation rule classes.
+The default rule classes (**Traditional Rules**) have the namespace ``CodeIgniter\Validation``,
+and the new classes (**Strict Rules**) have ``CodeIgniter\Validation\StrictRules``, which provide strict validation.
+
+The **Traditional Rules** implicitly assume that string values are validated.
```

Comment:
What do you mean by "implicitly assume that string values are validated"? I think this needs to be rephrased.

Reply:
You can also add a note that it uses PHP's type coercion.

Reply:
I mean the Traditional Rules implicitly assume to validate `$_POST`. 
The value type is only `string` or `string[]`.